zulipterminal/ui_tools/views.py

```diff
@@ -1307,6 +1307,20 @@ def __init__(self, controller: Any, stream_id: int) -> None:
         self.controller = controller
         stream = controller.model.stream_dict[stream_id]
 
+        stream_posting_policy = {
+            1: "Any user can post",
+            2: "Only owners and admins can post",
+            3: "Only members can post",
+            4: "Only owners, admins and moderators can post",
+        }
+        if stream.get("stream_post_policy") is None:
+            if stream.get("is_announcement_only"):
+                stream_policy = "Only admins can post"
+            else:
+                stream_policy = "Any user can post"
+        else:
+            stream_policy = stream_posting_policy[stream.get("stream_post_policy")]
```

Comment:
This is what I meant about the start of the other PR code, where there is something simpler.

Reply:
I will go through it once again and make the required changes.